### PR TITLE
Fix typos in documentation

### LINKED ISSUE #6484: Documentation: typo in several places, "exists" instead of "exits"
In TaskCluster documentation (eg [here](https://docs.taskcluster.net/docs/reference/workers/docker-worker/payload)), the word "exits" is written with a typo in several places

### PR DESCRIPTION
**Pull request by titooan**
Github Bug/Issue: Fixes #6484

### CODE REVIEW

**Comment by matt-boris** [comment]
Nice catch, thanks! I'm seeing this string appear in multiple of our schema files, not just the one you've fixed. Mind fixing those cases as well? 🙏🏻

**Comment by matt-boris** [comment]
And once you have the schemas fixed, you can run `yarn generate` to update auto-generated files that reference these schemas.

Also, will you please add a changelog using `yarn changelog` (level can be `silent`).

File: dev-docs/development-process.md [hunk 1]
```diff
@@ -206,7 +206,7 @@ Generation requires database to be running in order to properly generate schema
 
 ```sh
 # start the database and initialize it
-docker compose up -d postgres pg_int_db
+docker compose up -d postgres pg_init_db
```
Comment (matt-boris) [inline dev-docs/development-process.md:209]: Nice, thanks for fixing this too! 🎉

**Review by matt-boris — APPROVED** [review]
LGTM! Nice spotting 🦅 👀  and thanks for fixing!

**Comment by titooan** [comment]
> LGTM! Nice spotting 🦅 👀 and thanks for fixing!

Thanks a lot for your help on this PR!

**Pull request merged**